internal/nginx/manager.go

```diff
@@ -270,7 +266,11 @@ func (lm *LocalManager) ClearAppProtectFolder(name string) {
 func (lm *LocalManager) Start(done chan error) {
 	glog.V(3).Info("Starting nginx")
 
-	cmd := exec.Command(lm.binaryFilename)
+	binaryFilename := nginxBinaryPath
```

Comment:
this repeats 4 times in the code - can it be a function?

Reply:
makes sense 👍